01 - Community/Obsidian Roundup/2022-08-27 Visual Notetaking Guides & Demo Vaults for Managers.md

```diff
@@ -0,0 +1,90 @@
+---
+link: https://www.obsidianroundup.org/2022-08-27/
+author: Eleanor Konik
+published: 2022-08-27T12:30:05
+publish: true
+---
+
+# 2022-08-27: Visual Notetaking Guides & Demo Vaults for Managers
+You can now push notes to Readwise's Reader app for improved spaced repetition review. You can also comment on the Roundup!
+
+## 
```

Comment:
I wonder why this is.

Reply:
It would appear that an empty header is present in the roundup post itself and is also seen in the Atom XML.